src/Propel/Generator/Command/GraphvizGenerateCommand.php

```diff
@@ -41,7 +41,7 @@ protected function configure()
     /**
      * {@inheritdoc}
      */
-    protected function execute(InputInterface $input, OutputInterface $output)
+    protected function execute(InputInterface $input, OutputInterface $output): int
```

Comment:
I am not sure about the typehints here, even if "only" protected.
I didn't add them in https://github.com/propelorm/Propel2/pull/1590/files to make possible extensions less likely to break at this point.

Given that this is going into a new alpha or BC breaking release now anyway, I guess I also don't have strong objections either.

Reply:
Fair point, but this command is a bad candidate for overriding as it contains all the logic in one method.

Also I remember having some issues when this wasn't present, the commit I introduced this was:

> 3328fe17a9ca58907f445f694c80b95ac4ce1d16 Symfony 5 enforces return types on execute method